awx/ui_next/src/components/Chip/ChipGroup.jsx

```diff
@@ -1,9 +1,15 @@
 import React, { useState } from 'react';
-import { number } from 'prop-types';
+import { number, bool } from 'prop-types';
 import styled from 'styled-components';
 import Chip from './Chip';
 
-const ChipGroup = ({ children, className, showOverflowAfter, ...props }) => {
+const ChipGroup = ({
+  children,
+  className,
+  showOverflowAfter,
+  displayAll,
```

Comment:
It looks like this will automatically display all chips if `showOverflowAfter` is not given. We may not need the `displayAll` prop here. (I could see an argument that this is confusing though, in which case I would suggest defaulting `showOverFlowAfter` to a reasonable number like 5)